WSLGd/main.cpp

```diff
@@ -186,6 +191,37 @@ try {
 
     SetupOptionalEnv();
 
+    // "ulimits -c unlimited" for core dumps.
+    {
```

Comment:
I don't think these scopes are buying you anything, no smart pointers inside going out of scope, can you clarify your intent?

Reply:
Right, I was thinking to enable this path only when certain environment set, thus bracket was left over from "if (getenv("WSL2_WSLG_ENABLE_CORE_DUMP")" or I might add that eventually.